python/AzureEP/README.md

```diff
@@ -0,0 +1,15 @@
+Since onnxruntime 1.16, AzureEP supports the running of a "hybrid" model combining two compoments:
+
+- An edge model that runs locally.
+- A proxy model talks to a remote endpoint hosted hosting a model as a service.
+
+The file implemented three ways to create such a "hybrid" model:
```

Comment:
The file can be implemented in 3 ways

Reply:
How about - "There are three ways to create such ..."?